egs/wsj/s5/steps/online/nnet2/train_ivector_extractor.sh

```diff
@@ -115,6 +116,10 @@ splice_opts=$(cat $srcdir/splice_opts)
 gmm_feats="ark,s,cs:apply-cmvn-online --config=$dir/online_cmvn.conf $dir/global_cmvn.stats scp:$sdata/JOB/feats.scp ark:- | splice-feats $splice_opts ark:- ark:- | transform-feats $dir/final.mat ark:- ark:- | subsample-feats --n=$subsample ark:- ark:- |"
 feats="ark,s,cs:splice-feats $splice_opts scp:$sdata/JOB/feats.scp ark:- | transform-feats $dir/final.mat ark:- ark:- | subsample-feats --n=$subsample ark:- ark:- |"
 
+## This adds online-cmvn in $feats, upon request (configuration taken from UBM),
+## ('online_cmvn_iextractor' marks that we added online_cmvn)
+[ $online_cmvn ] && { feats="$gmm_feats"; touch $dir/online_cmvn_iextractor; }
```

Comment:
if  the variable is false, I think you should remove it just in case the dir was used before. e.g. `rm foo 2>/dev/null || true`.  Also please don't use brackets around $online_cmvn; it's true or false so it doesn't need them, and they are confusing.